stripper/ze_legoland_crackheads_p2.cfg

```diff
@@ -0,0 +1,40 @@
+;Fix boss from dieing on different endings which can cause a nuke bug
+modify:
+{
+	match:
+	{
+		"classname" "logic_auto"
+		"hammerid" "357281"
+	}
+	insert:
+	{
+		"OnMapSpawn" "ColorBoss,Kill,,0,-1"
+	}
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "trigger_push"
+		"hammerid" "45345"
+	}
+	insert:
+	{
+		"OnStartTouch" "ColorBoss,SetDamageFilter,,0,-1"
```

Comment:
SetDamageFilter isn't doing anything since the ColorBoss has no damage filter in the first place, not to mention the entity no longer exists at all because you're immediately killing it above?